.github/workflows/integration-tests.yml

```diff
@@ -35,14 +31,9 @@ jobs:
 
   dispatch-trivy:
     needs: [ build ]
-    runs-on: ubuntu-latest
-
-    steps:
-      - name: Fire off Trivy action
-        uses: broadinstitute/workflow-dispatch@v1
-        with:
-          workflow: Trivy
-          token: ${{ secrets.BROADBOT_TOKEN }}
+    uses: ./.github/workflows/trivy.yml
+    secrets: inherit
+    if: github.event_name == 'pull_request'
```

Comment:
Why are you now limiting this to pull requests?

Reply:
Bad copy pasta :'(